### PR TITLE
RF: moving nib-ls to a new cmdline folder

### PR DESCRIPTION
**Pull request by chrispycheng**
moved nib-ls to make generic use of its functions possible for a "nib-diff" function currently in progress. Shoutout to @yarikoptic

### CODE REVIEW

**Comment by matthew-brett** [comment]
I hate to say this, but it would be good to have a few little tests in the new module, if only as a template for adding more later.

**Comment by chrispycheng** [comment]
@matthew-brett can I ask what tests you're looking for? nib-ls functionality?

**Comment by matthew-brett** [comment]
Sorry - I meant tests for the not-main functions that are now in the `nibabel/cmdline` tree.

**Comment by matthew-brett** [comment]
Tests failing here - I think you need to add `nibabel.cmdline` to the list of packages in `setup.py`.  You should also add `nibabel.cmdline.tests` (does it have an `__init__.py` already.  There are some extra blank lines at the top of `cmdline.__init__.py`.  Otherwise, good to go (when tests running and passing).

**Comment by chrispycheng** [comment]
More tests to come for ap() and safe_get() functions

**Comment by matthew-brett** [comment]
Great, it will be good to have some more tests.

There are some PEP8 fails - see https://travis-ci.org/nipy/nibabel/jobs/351289012#L1492

**Comment by chrispycheng** [comment]
@matthew-brett Hi - if you have the time, could you please let me know what you think of the tests I've written? Thanks very much :-)

File: nibabel/cmdline/tests/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+class TestObject:
+    def __init__(self, test=None):
+        self.test = test
+
+    def get_test(self):
+        return self.test
+
+test = TestObject()
+test.test = 2
```
Comment (yarikoptic) [inline nibabel/cmdline/tests/test_utils.py]: please move it into `test_safe_get` so its purpose is clear and it doesn't get picked up by nose etc

Comment (matthew-brett) [reply nibabel/cmdline/tests/test_utils.py]: Yes, good idea.

**Comment by matthew-brett** [comment]
Yes, I agree - we can merge and improve.  @chrispycheng - do you mind making the change Yarik suggested, and I will merge.

File: nibabel/cmdline/tests/test_utils.py (hunk 1 above)
Comment (chrispycheng) [reply nibabel/cmdline/tests/test_utils.py]: done and ready to rumble

**Pull request merged**